test/controllers/companies_controller_test.rb

```diff
@@ -2,8 +2,8 @@
 
 class CompaniesControllerTest < ActionController::TestCase
   setup do
-    @employee = Employee.create(email: 'user@example.com', password: 'foobar')
     @company = Company.create(name: 'foo-llc')
+    @employee = Employee.create(email: 'user@example.com', password: 'foobar', company_id: @company.id)
     @request.env["devise.mapping"] = Devise.mappings[:employee]
```

Comment:
Some of the tests were failing because the employee we have logged in didn't belong to a company.